packages/sit-onyx/src/components/OnyxFlyoutMenu/OnyxFlyoutMenu.stories.ts

```diff
@@ -0,0 +1,43 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxFlyoutMenu from "./OnyxFlyoutMenu.vue";
+import { h } from "vue";
+import OnyxFlyoutMenuItem from "../OnyxFlyoutMenu/OnyxFlyoutMenuItem.vue";
+
+const meta: Meta<typeof OnyxFlyoutMenu> = {
+  title: "support/OnyxFlyoutMenu",
```

Comment:
```suggestion
  title: "support/FlyoutMenu",
```
We don't use the Onyx prefix here :)

Reply:
fixed